README.md

```diff
@@ -53,7 +55,20 @@ The assets are stored in the following directories (relative to your installatio
  - aw-server-python: `activitywatch/aw-server/aw_server/static/`
  - aw-server-rust: `activitywatch/aw-server-rust/static/`
 
-Once you've put the files in the directories, you may have to do a hard refresh in your browser to invalidate any stale caches.
+`cd aw-server && make build` will copy the build assets from aw-webui for you. Once you've put the files in the directories, you may have to do a hard refresh in your browser to invalidate any stale caches.
```

Comment:
> `cd aw-server && make build` will copy the build assets from aw-webui for you.

This is not true if you're not running aw-server from a release/install, nor if you've fetched the latest build of aw-webui from CI, nor if you're using aw-server-rust (I think?).

```suggestion
Either copy the assets manually, or run `make build` from the `aw-server` parent directory to rebuild and copy the assets for you.

Once you've put the files in the directories, you may have to do a hard refresh in your browser to invalidate any stale caches.
```